quill-sql/src/test/scala/io/getquill/context/sql/norm/ExpandNestedQueriesSpec.scala

```diff
@@ -28,4 +29,18 @@ class ExpandNestedQueriesSpec extends Spec {
     testContext.run(q).string mustEqual
       "SELECT x.* FROM (SELECT b.i FROM TestEntity a, TestEntity2 b) x LIMIT 10"
   }
+
+  "tokenize property" in {
+    object testContext extends SqlMirrorContext[MirrorSqlDialect, SnakeCase]
+    import testContext._
+
+    case class Entity(camelCase: String)
+
+    testContext.run(
+      query[Entity]
+      .map(e => (e, 1))
+      .nested
+    ).string mustEqual
+      "SELECT x.camelCase, x._2 FROM (SELECT e.camel_case camelCase, 1 _2 FROM entity e) x"
```

Comment:
Cool - I didn't have reason to believe it wasn't, just wanted to call out the question since columns are usually selected as whatever case the column name is